networktest.go

```diff
@@ -154,7 +155,7 @@ func newLightningNode(btcrpcConfig *rpcclient.ConnConfig, lndArgs []string) (*li
 	return &lightningNode{
 		cfg:               cfg,
 		p2pAddr:           net.JoinHostPort("127.0.0.1", strconv.Itoa(cfg.PeerPort)),
-		rpcAddr:           net.JoinHostPort("127.0.0.1", strconv.Itoa(cfg.RPCPort)),
+		rpcAddr:           net.JoinHostPort(cfg.RPCHost, strconv.Itoa(cfg.RPCPort)),
```

Comment:
I don't think this is necessary. This is integration testing infrastructure so all communication should be performed over local host.